README.md

```diff
@@ -43,6 +43,12 @@ npm install babel-cli
 > Note for npm 2 users, you have to make sure po2json in the root `node_modules`, the same way as
 `babel-cli`
 
+And finally, install `babel-runtime` in your dev-dependencies needed to be able to compile the code.
```

Comment:
`babel-runtime` is not compiling anything, it will be required though at "runtime" if code was compiled to leverage it. (which is the case with our preset).